### ISSUE TITLE
Gather statistics on "manual email confirmations"

### ISSUE THREAD

**Issue by WilliamWDTK** [mysociety/whatdotheyknow-theme#1650]
Sometimes we receive emails in the support inbox where users reply to our emails that ask them to click a link to confirm their account etc by saying "Yes, I confirm".  We categorise and process these as "manual email confirmations". However, sometimes the account has already been confirmed, which suggests that the link has been clicked. This would normally be by the user, unless there is a collision between 2 or more admins trying to deal with the thread at the same time.

It might be useful to collect data on how often the account has been confirmed by clicking the link or not when we receive these emails.

**Comment by HelenWDTK** [comment]
I'd like to experiment with changing the subject lines on these, as I suspect that may be what is causing the replies.

**Comment by mdeuk** [comment]
> I'd like to experiment with changing the subject lines on these, as I suspect that may be what is causing the replies.

I pondered that. If we look at the example of the HM Government / Parliament petitions site, they use a similar format…

```
Subject: Please confirm your email address

Click this link to sign the petition "[petition name]"

https://petition.parliament.uk/signatures/[ID]/verify?token=[ID]

Thanks,
The Petitions team
UK Government and Parliament
```

Interestingly, [despite being in HTML _and_ text](https://github.com/alphagov/e-petitions/tree/master/app/views/petition_mailer), there isn't a "button" as you might have expected.

Is our problem, perhaps, that the message is a bit too wordy, so the CTA gets missed?

**Comment by HelenWDTK** [comment]
> Is our problem, perhaps, that the message is a bit too wordy

Possibly. I think a lot of them seem to be sent from mobile, so my theory was that users might be seeing a notification and hitting reply without reading the email at all. It could be that the link is just too hard to spot.

**Comment by WilliamWDTK** [comment]
>I think a lot of them seem to be sent from mobile

I think that could well be the case.  The other thing that could (and does) happen on mobile is that there is a list of 3 or so suggested replies, of which the common "I confirm" could be one.  This sometimes appears in the notification itself for me, though I've not noticed that with email, only certain IM platforms, other times the replies are suggested in-app.

**Comment by WilliamWDTK** [comment]
I've created a couple of sub-labels for tracking whether the email address had already been confirmed when checking in the admin interface.

label | count
-- | --
Help/manual email confirmation | 23 conversations
…/already confirmed | 1 conversation
…/confirmed by admin | 0 conversations
…/instructed user to confirm | 0 conversations

**Comment by HelenWDTK** [comment]
I've noticed that FMS sends the "Confirm your report on FixMyStreet" emails from a no reply address, so that's one option. I still think tweaking the subject line will be the answer though. 

I've found the right bit of code to change to make this happen - does anyone have any thoughts about what we should try?

**Comment by WilliamWDTK** [comment]
Perhaps "verify" instead of "confirm", which has less of a suggestion of entering into correspondence?

Or even, "click the link to verify your email address"

**Comment by garethrees** [comment]
Linking to https://github.com/mysociety/whatdotheyknow-private/issues/219#issuecomment-1514558483

**Comment by HelenWDTK** [comment]
I've been digging in the code and we actually send a lot of "confirm" emails:

```
Confirm your account on {{site_name}}
Confirm your new email address on {{site_name}}
Confirm you want to follow the request {{request_title}}
Confirm you want to follow new requests
Confirm you want to follow all successful FOI requests
Confirm you want to follow requests to {{public_body_name}}
Confirm you want to follow requests by {{user_name}}
Confirm you want to follow new requests or responses matching your search
Confirm your FOI request to {{public_body_name}}
```

**Comment by garethrees** [comment]
Just dropping some notes from a chat about how our other services handle this.

FMS identifies replies to auto-generated emails, and then auto-responds to them.

* https://github.com/mysociety/fixmystreet/blob/master/bin/handlemail is script forwarded from .forward
* https://github.com/mysociety/fixmystreet/blob/master/perllib/FixMyStreet/Email/Incoming.pm is the actual logic code
* https://github.com/mysociety/fixmystreet/blob/master/templates/email/default/reply-autoresponse is the default autoresponse email

WTT has a similar process (https://github.com/mysociety/writetothem/blob/master/bin/handlemail). If a user then replies to the auto-reply, it goes through to support.

Anecdotally, not much changed with wording tweaks to the emails we send.

**Comment by garethrees** [comment]
Linking to https://github.com/mysociety/whatdotheyknow-private/issues/219
